charmcraft.yaml

```diff
@@ -3,23 +3,7 @@ parts:
   charm:
     source: .
     plugin: reactive
-    build-snaps:
-      # temporary fix because charm cannot install/build on 20.04 using
-      # charm/3.x without unnecessary dependency in wheelhouse.txt:
-      # Jinja2<3;python_version == '3.8'
-      # setuptools~=46.0.0;python_version == '3.8'
-      - charm/2.x/stable
-bases:
-    - build-on:
-        - name: ubuntu
-          channel: "22.04"
-          architectures: ["amd64"]
-      run-on:
-        - name: ubuntu
-          channel: "22.04"
-          architectures:
-              - amd64
-        - name: ubuntu
-          channel: "20.04"
-          architectures:
-              - amd64
+    build-snaps: [charm/2.x/stable]
```

Comment:
Can we try dropping this pin and updating dependencies as required as part of this update?

Reply:
it could be possible, we need to check what dependency can be unpinned now

Reply:
this seems work